### DISCUSSION TITLE
Rodentcam: spend $20 to watch rats eat your tomatoes

### DISCUSSION THREAD

**Discussion by tve** [blakeblackshear/frigate#1867] [General]
One of the applications I have for Frigate is to find out who eats our veggies in the garden. Yesterday I set up the first camera under some tall tomato vines that have been attacked recently. Lo and behold, this morning Frigate greeted me with 4 events! Here's a quick snap of one:
![Screenshot from 2021-09-28 08-53-28](https://user-images.githubusercontent.com/39480/135122653-835ee787-6835-4921-92de-d0d2b77d8680.png)
The little rat got identified 3x as cat (hilarious), once as bird, and once as person.
The camera is an esp32-cam with a 160 degree lens from which I removed the IR filter (super easy). It's stuck into a cheap Hammond box and held on a little table-top tripod. I used a cheap IR illuminator which I hot-glued on top. Total cost a little over $20 :-).
BTW: Yes, the rascal triggered one of the traps but didn't get caught. After seeing the action I have some ideas about how to better position the traps.

**Comment by ragenhe** [comment]
@tve - If I may ask, what firmware are you running on the ESP32-CAM?

**Reply by tve** [reply]
https://github.com/easytarget/esp32-cam-webserver
I really like that I can tweak the config via web interface. I believe some of the others may produce a slightly higher frame rate, at least some of my tests seemed to show that. I'm mainly using XGA and HD frame sizes depending on which of the two aspect ratios works best. If you want UXGA resolution you need to tweak the driver a bit to increase DMA buffers and that's a pain, but you also need to get a proper lens else the extra res only gives you a better view of the lens blur and distortion :-).

**Reply by ragenhe** [reply]
Thanks, I will check it out!
